sysutils/portfile-gen/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        macports macports-contrib 5447906b1c3be5b6189e61c6256dc75c291c59e5
+github.tarball_from archive
+
+name                portfile-gen
+version             0.2
+revision            0
+categories          sysutils macports
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           any
+license             BSD
+supported_archs     noarch
+
+description         Generate a basic template Portfile given a few bits \
+                    of information
+long_description    {*}${description}.
+homepage            https://github.com/${github.author}/${github.project}/blob/master/${name}
+
+livecheck.type      none
```

Comment:
why would you switch off the livecheck?

Reply:
Probably a leftover from when I couldn't get it to work previously; removed in 7907811